pkcs8/src/common.rs

```diff
@@ -0,0 +1,76 @@
+use core::convert::TryFrom;
```

Comment:
I'd probably suggest making this module `version.rs`, and splitting the attributes-related code into `attributes.rs`.